subprojects/code-quality/src/integTest/groovy/org/gradle/api/plugins/quality/checkstyle/CheckstylePluginVersionIntegrationTest.groovy

```diff
@@ -262,6 +266,26 @@ class CheckstylePluginVersionIntegrationTest extends MultiVersionIntegrationSpec
         executedAndNotSkipped(":checkstyleMain")
     }
 
+    @IgnoreIf({ !isSarifSupported(version.toString()) })
```

Comment:
:x: `version` will not be initialized when the annotation parameter will be evaluated.

You should instead do the following at the begining for the test method implementation:

```groovy
assumeTrue(isSarifSupported(version.toString())
```